File: cases/taint/python/local-overwrite-negative/local_overwrite.py
def dfb_source():  # DFB-SOURCE: local-overwrite-input
    return "tainted"


def dfb_sink(value):  # DFB-SINK: local-overwrite-sink
    pass


def run():
    value = dfb_source()
    value = "clean"  # DFB-KILL: local-overwrite-clean
    dfb_sink(value)

File: cases/taint/ruby/local-overwrite-negative/local_overwrite.rb
def dfb_source # DFB-SOURCE: local-overwrite-input
  "tainted"
end

def dfb_sink(value) # DFB-SINK: local-overwrite-sink
end

def run
  value = dfb_source
  value = "clean" # DFB-KILL: local-overwrite-clean
  dfb_sink(value)
end
